### PR TITLE
Event style redesign

### PR DESCRIPTION
**Pull request by mrmaDrma**
I accidentally closed Pull Request #55 
So I've fixed everything from feedbacks on old one and created new one

This Pull Request resolves issue [here](https://github.com/big-neon/bn-mobile-react/issues/56)

### CODE REVIEW

**Comment by mileTadic1994** [comment]
[BigNeon.mp4.tar.gz](https://github.com/big-neon/bn-mobile-react/files/3236989/BigNeon.mp4.tar.gz)

Here is short demo of new page look. I had to zip it, since .mp4 is not supported format here. 



> Here is [BigNeon.mp4.tar.gz](https://github.com/big-neon/bn-mobile-react/files/3237023/BigNeon.mp4.tar.gz) short demo of redesigned event view.
> 
> And couple of screenshots
> 
> screen1
> screen2

File: src/LinearGradient.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { LinearGradient } from 'expo'
```
Comment (simple-taskic) [inline src/LinearGradient.js]: I really like this approach as we have expo wrapped here.
Please, can you try to find LinearGradient import from some other resources (not expo) as we will need to move away from expo?

Comment (mileTadic1994) [reply src/LinearGradient.js]: I've found this library https://github.com/react-native-community/react-native-linear-gradient,
but I couldn't make it run with expo. Maybe we should try use it when we completely move from expo ?

Comment (simple-taskic) [reply src/LinearGradient.js]: Yup, it won't work with expo. With expo only works when you have just JS implementation.

**Comment by simple-taskic** [comment]
Hey, @mileTadic1994 did you manage to address all comments from this PR?

**Comment by mileTadic1994** [comment]
> Hey, @mileTadic1994 did you manage to address all comments from this PR?

Hey, @simple-taskic, @Krakaw, yeah I've just updated everything, except this part where we need to remove Expos Linear Gradient. Should we wait to remove Expo first?

**Comment by simple-taskic** [comment]
@mileTadic1994 I resolved that in `eject from expo` branch, so you are good to go then.
@Krakaw you can merge this one if you think it's ready.

**Comment by Krakaw** [comment]
@zmelamed I'm happy from an implementation side. Please can you confirm from the design side.

**Comment by Krakaw** [comment]
Closing this as we are moving forward with new designs.

**Pull request closed without merging**